pkg/util/proto/workloadmeta.go

```diff
@@ -419,6 +421,53 @@ func WorkloadmetaFilterFromProtoFilter(protoFilter *pb.WorkloadmetaFilter) (*wor
 	return workloadmeta.NewFilter(kinds, source, eventType), nil
 }
 
+func toLanguage(proto *pb.Language) *languagemodels.Language {
+	if proto == nil {
+		return nil
+	}
+	return &languagemodels.Language{
+		Name: languagemodels.LanguageName(proto.GetName()),
+	}
+}
+
+// WorkloadmetaEventFromProcessEventSet converts the given ProcessEventSet into a workloadmeta.Event
```

Comment:
Given that's not generic code, I'm not sure it should be in this package, perhaps more suited in the `process_collector`